.github/workflows/client.yml

```
@@ -34,3 +33,28 @@ jobs:
       - run: cd client/ionic && npm install
       - run: cd client/ionic && npm audit --audit-level=moderate
       - run: cd client/ionic && CI=true npm test
+  # This builds the android APK
+  build-android-debug:
+    strategy:
+      matrix:
+        platform: [ubuntu-latest, macos-latest]
```

Was there a particular reason why you're running this both on Ubuntu and MacOS? I'd imagine we'd add another job for building the iOS version that would have to run on MacOS, anyway.

```suggestion
        platform: [ubuntu-latest]
```

---

Yes, we want to make sure that tests run on both environments.